.github/ISSUE_TEMPLATE/dep-monthly.md

```diff
@@ -1,8 +1,8 @@
 ---
-name: Monthly Run DEP CATS permits
+name: [publish] Monthly Run DEP CATS permits
 title: dep_cats_permits
```

Comment:
should `[publish]` be in the title rather than the name? looks like that's the case in `build-publish.md` and the `publish.yml` action is looking at `github.event.issue.title`

Reply:
Yes! Good catch.